lib/workload/stateless/stacks/workflow-manager/workflow_manager/tests/factories.py

```diff
@@ -0,0 +1,59 @@
+from enum import Enum
+import uuid, json
+from datetime import datetime
+from zoneinfo import ZoneInfo
+
+import factory
+from django.utils.timezone import make_aware
+
+from workflow_manager.models.workflow import Workflow
+from workflow_manager.models.workflow_run import WorkflowRun
+from workflow_manager.models.payload import Payload
+
+
+class  TestConstant(Enum):
+    workflow_name = "TestWorkflow1"
+    payload = {
+        "key": "value",
+        "foo": uuid.uuid4(),
+        "bar": datetime.now().astimezone(ZoneInfo('Australia/Sydney')),
```

Comment:
This zone info should work, right? Perhaps, we should run `python manage.py generate_mock_data` locally to confirm it.

Reply:
Yes, that should work. 
Example mock generation work with [5afd11a](https://github.com/umccr/orcabus/pull/325/commits/5afd11af1e5b14e4ef0feefb3618fbfff5fe35f6)